test_data/gomod_packages.yaml

```diff
@@ -0,0 +1,110 @@
+# gomod package without dependencies
```

Comment:
Could we move this directory to tests/integraion/data or tests/integration/test_data?